commands.js

```diff
@@ -2007,6 +2007,7 @@ exports.commands = {
 	blacklist: function (target, room, user) {
 		if (!target) return this.parse('/help blacklist');
 		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (target === 'show') return this.parse('/showblacklist');
```

Comment:
I was more implying that this user hasn't been on PS in quite some time, so the chance of them just showing up and randomly causing trouble seems to be unlikely to me